client/src/components/Navbar/Logo.tsx

```diff
@@ -0,0 +1,24 @@
+import { Grid, Typography, makeStyles } from '@material-ui/core';
+import { useHistory } from 'react-router';
+
+const useStyles = makeStyles({
+  text: {
+    letterSpacing: 8,
+    color: 'white',
+    fontWeight: 'bolder',
+    fontSize: 20,
+  },
+});
```

Comment:
let's move these to a style file outside of the main tsx to be consistent with the other files. We can discuss in standup whether we want to rename the style files to be more readable